src/patternfly/components/VerticalNav/styles.scss

```diff
@@ -0,0 +1,134 @@
+@import "../../patternfly-utilities";
+
+.pf-c-vertical-nav {
+  /* stylelint-disable */
+
+  // Vertical nav
+  --pf-c-vertical-nav--Color:             var(--pf-global--BackgroundColor--dark-100);
```

Comment:
Please let's not use a BackgroundColor to define a color. If we need more color options on a global level let me know